### PR TITLE
test/ceph-fio: add ceph test script and fio configuration

### PR DESCRIPTION
**Pull request by CongMinYin**
test ceph cluster performance including iops, band width, latency

Signed-off-by: Yin Congmin <congmin.yin@intel.com>

### CODE REVIEW

**Comment by changchengx** [comment]
@CongMinYin 
Please try to add License and Copyright info in source file header.
Example: https://github.com/dcperf/cloudbench/blob/master/test/sysbench/mysql/sysbench_write.sh#L2

**Comment by changchengx** [comment]
@CongMinYin 
In "pull-request title" or "commit title" or "commit message", please give info what the testing purpose instead of just "ceph test script"

File: test/ceph-fio/README [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Document explain:
+
+fio-init.conf: Before test, exec"fio fio-init.conf" to write 8 images.
+fio-env.conf: It's a fio conf, using env parameter to decide FIO_SIZE, FIO_DEPTH, FIO_JOBS.
```
Comment (changchengx) [inline test/ceph-fio/README:4]: Please rename the file to fio-env.conf instead of fio-env.sh

File: test/ceph-fio/README [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+test-env.sh: test script, modify the arry to test what you need.
+item.sh: print test item, such as "4k-8depth"
```
Comment (changchengx) [inline test/ceph-fio/README]: Please align the content in README with the right file name

File: test/ceph-fio/README [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+iops.sh, bw.sh, lat.sh, taillat.sh: Extract the data from the test results and print it out. For example, iops.sh will print all the iops from all "*.txt" in order.
```
Comment (changchengx) [inline test/ceph-fio/README]: Check the above comment.

File: test/ceph-fio/README (hunk 1 above)
Comment (CongMinYin) [reply test/ceph-fio/README:4]: done

**Comment by CongMinYin** [comment]
@changchengx  done. I add copyright in all *.sh except *.conf and README, do these need?

**Comment by changchengx** [comment]
@CongMinYin 
You can add the license header in *.conf file.
Currently, we don't need to add the license header in README or other data documents.
You can refer https://spdx.org/licenses/ for more info.

**Comment by changchengx** [comment]
@CongMinYin 
Please rebase your code repository to the master branch and then submit the PR.
There're two commits in this PR, one of the commits is not needed.

**Comment by changchengx** [comment]
I'll merge this PR after adding the license header.
Example: https://github.com/dcperf/cloudbench/blob/master/test/fio/libaio/fio_run_rbd_vm_disk_70r_30w.sh#L2

**Comment by CongMinYin** [comment]
> I'll merge this PR after adding the license header.
> Example: https://github.com/dcperf/cloudbench/blob/master/test/fio/libaio/fio_run_rbd_vm_disk_70r_30w.sh#L2

I have add the license header.

**Pull request merged**